### ISSUE TITLE
Duplicate describe title let reporter crash

### ISSUE THREAD

**Issue by christian-bromann** [webdriverio/webdriverio#1621]
## The problem

When the user has multiple describe blocks with the same title like

``` js
describe('automated builds tab multiple jobs', () => {
  before(() => {
    // ...

describe('automated builds tab multiple jobs', () => {
  before(() => {
    //
```

the reporter fails like:

```
00:33:58 /home/sauce/workspace/jam-protractor/node_modules/webdriverio/build/lib/utils/ReporterStats.js:355
00:33:58             if (!suiteStats.tests[title]) throw Error('Unrecognised test [' + title + '] for suite [' + runner.parent + ']');
00:33:58                                           ^
00:33:58 
00:33:58 Error: Unrecognised test [should filter by owner] for suite [automated builds tab multiple jobs]
00:33:58     at Error (native)
00:33:58     at ReporterStats.getTestStats (/home/sauce/workspace/jam-protractor/node_modules/webdriverio/build/lib/utils/ReporterStats.js:355:49)
00:33:58     at ReporterStats.output (/home/sauce/workspace/jam-protractor/node_modules/webdriverio/build/lib/utils/ReporterStats.js:363:22)
00:33:58     at BaseReporter.<anonymous> (/home/sauce/workspace/jam-protractor/node_modules/webdriverio/build/lib/utils/BaseReporter.js:115:25)
00:33:58     at emitOne (events.js:77:13)
00:33:58     at BaseReporter.emit (events.js:169:7)
```
## Environment
- WebdriverIO version: latest
- [Standalone mode or wdio testrunner](http://webdriver.io/guide/getstarted/modes.html): wdio
- if wdio testrunner, running synchronous or asynchronous tests: sync
- Additional wdio packages used (if applicable): spec reporter, mocha framework

**Comment by janmeier** [comment]
This also seems to affect tests with multiple capabilities (where the same describe also appears several times, but for different browsers)

**Comment by christian-bromann** [comment]
@janmeier do you have an example?

**Comment by janmeier** [comment]
wdio.conf
```js
capabilities: [{
    browserName: 'chrome'
  }, {
    browserName: 'safari',
    version: 9,
    platform: 'OS X 10.11'
  }],
```

```js
describe('organization influencers', function () {
  it('should be possible to invite an influencer', ...);
})
```
```
------------------------------------------------------------------
[safari 9 OS X 10.11 #1c] Session ID: 3286079d36cf498890cb234e753649bc
[safari 9 OS X 10.11 #1c] Spec: /home/ubuntu/frontend/test/organization/influencer.test.js
[safari 9 OS X 10.11 #1c] Running: safari (v9) on OS X 10.11
[safari 9 OS X 10.11 #1c]
[safari 9 OS X 10.11 #1c]   organization influencers
[safari 9 OS X 10.11 #1c]       1) should be possible to invite an influencer
[safari 9 OS X 10.11 #1c]
[safari 9 OS X 10.11 #1c]
[safari 9 OS X 10.11 #1c] 1 failing (1m, 21s)
[safari 9 OS X 10.11 #1c]
[safari 9 OS X 10.11 #1c] 1) organization influencers should be possible to invite an influencer:
[safari 9 OS X 10.11 #1c] Promise was rejected with the following reason: Error: An element could not be located on the page using the given search parameters.
[safari 9 OS X 10.11 #1c] Error: Promise was rejected with the following reason: Error: An element could not be located on the page using the given search parameters.
[safari 9 OS X 10.11 #1c]     at elements("h1=Influencers") - getText.js:18:17
[safari 9 OS X 10.11 #1c]     at getText("h1=Influencers") - waitForText.js:37:22
[safari 9 OS X 10.11 #1c]

/home/ubuntu/frontend/node_modules/webdriverio/build/lib/utils/ReporterStats.js:403
            testStats.state = 'fail';
                            ^

TypeError: Cannot set property 'state' of undefined
    at ReporterStats.testFail (/home/ubuntu/frontend/node_modules/webdriverio/build/lib/utils/ReporterStats.js:403:29)
    at BaseReporter.<anonymous> (/home/ubuntu/frontend/node_modules/webdriverio/build/lib/utils/BaseReporter.js:159:25)
    at emitOne (events.js:96:13)
    at BaseReporter.emit (events.js:188:7)
    at BaseReporter.handleEvent (/home/ubuntu/frontend/node_modules/webdriverio/build/lib/utils/BaseReporter.js:300:27)
    at Launcher.messageHandler (/home/ubuntu/frontend/node_modules/webdriverio/build/lib/launcher.js:594:28)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at handleMessage (internal/child_process.js:718:10)
    at Pipe.channel.onread (internal/child_process.js:444:11)
```

**Comment by christian-bromann** [comment]
This should be get fixed by #1717
